ts/analysis-component.ts

```diff
@@ -69,8 +70,8 @@ export class AnalysisComponent {
                     'First': utils.aggregators['First'],
                     'Last': utils.aggregators['Last']
                 },
-                rows: ['lem1'],
-                cols: ['pos1'],
+                rows: defaultVariable.map(v => `lem_${v}`),
+                cols: defaultVariable.map(v => `pos_${v}`),
                 renderer: heatmap,
```

Comment:
What happens if defaultVariable = []? Are rows and cols then undefined? If so, is this the intended behaviour?

Reply:
Yes, the table will be empty then. The purpose of this is to show the currently selected rows and columns of the pivot table. The user can always adjust add/remove (meta) variables from the interface.